.travis.yml

```diff
@@ -83,16 +50,8 @@ jobs:
         - docker tag outline/shadowbox quay.io/outline/shadowbox:daily
         - docker push quay.io/outline/shadowbox:daily
 
-    - stage: deploy
-      name: Manager Linux
-      addons:
-        apt:
-          packages:
-          - rpm
-      script: yarn do server_manager/electron_app/package_linux
```

Comment:
Do you have thoughts about what to do with these package_<env> actions?

I'm wondering if they're adding value or can be removed? Ideally the packaging action would catch breakages before we try to release, but I'm curious if that's ever happened?

Maybe we can move the package_<env> actions to our daily build?

Reply:
I think we should run them in the push tests, to make sure packaging is not broken. There are many things that can break it, as you saw in the Polymer 3 PR.

Reply:
I don't think we would have caught that bug just by building the packages though, that was a runtime problem.  We could write an integration test to fix that though.